### ISSUE TITLE
BootCamp requires windows 7

### ISSUE THREAD

**Issue by Tracker-Friendly** [timsutton/brigadier#70]
I have tried to download the latest drivers (on windows 10) for the macbook air mid 2012 (MacBookAir5,2) but the installer specified that it required windows 7 despite being a 6x version. I attempted to use an older version (by seeing the supported hardware list and use -m to specify an older model to download it) and it installed, however, it would not let me right click drag, a common issue on older versions. Please help me to be able to get the windows 10 version, or tell me how to be able to right click drag. (P.S I have tried TrackPad++ but it bricked my computer by messing up my drivers).

**Comment by forvenus** [comment]
same problem here, macbook pro 15 2012(MacBookPro10,1)

**Comment by CatsLover2006** [comment]
I have the opposite problem, MacbookPro9,1 and I have several Windows issues
1) I have to disable the integrated GPU to prevent system-crippling BSODs
2) Can't even download the "latest" support software
3) My current downloaded boot camp software, a 5.x version, installs but immediately locks up the computer after a reboot

**Comment by Arzumify** [comment]
Lol, i ended up installing arch linux. Good luck with your issue.

**Comment by CatsLover2006** [comment]
Fair enough, I could do that but I wanna be able to use Windows for gaming.

**Comment by d1stgman** [comment]
I had this issue too and found a work around. Right click the setup file in the bootcamp folder and go to properties, Click on the Compatibility tab on top, Check Run this program in compatibility mode for and select windows 8 (it was the newest version available for me).

I am in a bootloop now though

**Comment by DevX-Cipher** [comment]
you need windows 10 drivers the script seems to be old and not getting updated anymore

**Comment by CatsLover2006** [comment]
Just downloaded the support software from Boot Camp itself.

**Comment by Arzumify** [comment]
What if people...

Don't want to install macOS?

I know crazy idea right?

**Comment by DevX-Cipher** [comment]
you dont have to i have macbook and it only has windows 11 on it

**Comment by jameswsullivan** [comment]
I ran into the same problem on a Mac Mini mid-2011 (Macmini5,1) using Brigadier. (Windows was natively installed via USB, no macOS.)

Got around by going [here](https://www.driverscape.com/manufacturers/apple/laptops-desktops/macmini5%2C1/16030) to directly download the needed drivers. Also you might wanna install Windows 10 using Legacy BIOS mode instead of UEFI to have less driver-related issues.

I wrote a blog post [here](https://jameswsullivan.github.io/refurbish-mac-mini-a1347-late-2014-and-mid-2011-part-4/) if it helps.

**Comment by DevX-Cipher** [comment]
i directly install windows 10 no mac osx at all

**Comment by Chozycode** [comment]
@d1stgman Genius, thanks a lot. the workaround was on point.

**Comment by skelllx** [comment]
> I ran into the same problem on a Mac Mini mid-2011 (Macmini5,1) using Brigadier. (Windows was natively installed via USB, no macOS.)
> 
> Got around by going [here](https://www.driverscape.com/manufacturers/apple/laptops-desktops/macmini5%2C1/16030) to directly download the needed drivers. Also you might wanna install Windows 10 using Legacy BIOS mode instead of UEFI to have less driver-related issues.
> 
> I wrote a blog post [here](https://jameswsullivan.github.io/refurbish-mac-mini-a1347-late-2014-and-mid-2011-part-4/) if it helps.

I found a better way to make everything work.
I also have a Mac Mini (Mid 2011), Model Mac Mini 5,1. I have made a custom Windows 10 Pro ISO installation file with all the drivers that come with Bootcamp ~ pre-installed for this Mac Mini in my custom Windows 10 iso. The Display, Internal Speakers, Audio over HDMI, Audio Ports (headphone and audio-in)/ USB/IR Sensor/Thunderbolt 1/FireWire/ SD Card/ Ethernet/ WiFi/ Bluetooth, all work properly and without any hassle.

Just needs Rufus for making it bootable with MBR partition.
I'll be making a Reddit Post soon and/or will attach the Custom Windows 10 Pro ISO here.
Edit : Here's the Direct [Link](https://drive.google.com/drive/folders/13z2IMcPApcJmvTovBsQkNz0AQ5FvRhmh?usp=sharing ) for the files. Please read the READ ME.txt and INFO.txt first.
